test/runner.tmpl

```diff
@@ -7,6 +7,14 @@
 		<link rel="stylesheet" type="text/css" href="/node_modules/mocha/mocha.css" />
 		<script src="/node_modules/mocha/mocha.js"></script>
 		<script src="/node_modules/chai/chai.js"></script>
+		<style>
+			/* This is essential to stop mocha reporter created anchors from making the document overflow, mainly along the x-axis */
+			#mocha h1 a {
+				overflow-wrap: break-word;
+  			word-wrap: break-word;
```

Comment:
Weird indentation here. Maybe we should run Prettier over our HTML too?

Reply:
This comment wasn't resolved.

Reply:
![image](https://user-images.githubusercontent.com/20978252/56734326-62928d80-675a-11e9-9e8f-59cc31bc26a6.png)

Github decides to show it this way 🤔